imports/client/components/GuessQueuePage.tsx

```diff
@@ -138,7 +210,7 @@ interface GuessQueuePageProps extends GuessQueuePageWithRouterParams {
 }
 
 const GuessQueuePage = (props: GuessQueuePageWithRouterParams) => {
-  useBreadcrumb({ title: 'Guess queue', path: `/hunts/${props.match.params.huntId}/guesses` });
+  useBreadcrumb({ title: 'Guess queue', path: `/ hunts / ${props.match.params.huntId} /guesses` });
```

Comment:
I haven't tested this, so correct me if I'm wrong, but I think this new whitespace here is going to do unwelcome things to the breadcrumb link

Reply:
I'm not sure where this whitespace came from. I think an over-zealous auto-formatter